test/app_automate_driver_test.rb

```diff
@@ -92,23 +92,38 @@ def test_wait_for_element_defaults
     driver = AppAutomateDriver.new(USERNAME, ACCESS_KEY, LOCAL_ID, TARGET_DEVICE, APP_LOCATION)
 
     mocked_element = mock('element')
-    mocked_element.expects(:displayed?).returns(true)
+    mocked_element.expects(:displayed?).returns(true).at_least_once
```

Comment:
Whats with all the at_least_once calls?

Reply:
It only actually needs them in the one scenario, fixed that.  It's necessary in the failure scenario as the Selenium::Wait will result in the `find_element` method being called multiple times, with the `displayed?` method being called each time as well.